src/components/Alert/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC, ReactNode, useState } from "react";
+
+export interface AlertPropType {
+  title?: string;
+  type?: "info" | "error" | "success" | "warning";
+  message: ReactNode;
+}
+
+const getStylesByType = (type: AlertPropType["type"]): string =>
+  [
+    "border px-5 py-3 bg-opacity-5",
+    "flex gap-x-5 pr-8 relative flex-wrap",
+    type === "info" && "border-blue bg-blue",
+    type === "error" && "border-error bg-error",
+    type === "success" && "border-green bg-green",
+    type === "warning" && "border-warning bg-warning",
+  ]
+    .filter(Boolean)
+    .join(" ");
+
+export const Alert: FC<AlertPropType> = ({ title, type = "info", message }) => {
+  const [isClosed, setIsClosed] = useState(false);
+  const styles = getStylesByType(type);
+  if (isClosed) return null;
+  return (
+    <div className={styles} role='alert'>
+      {title && <h4 className='font-bold'>{title}</h4>}
+      <p>{message}</p>
+      <button
+        className={[
+          "text-blue absolute top-2 right-3 transform",
+          "hover:text-purple transition hover:rotate-180 p-2 rounded-full focus-offset",
+        ].join(" ")}
+        onClick={() => setIsClosed(true)}
+      >
+        <svg width='16' height='16' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
I'm wondering if it makes sense to create a dedicated SVG for the X icon. We'll probably need it in other places as well